tracing-subscriber/src/filter/mod.rs

```diff
@@ -11,10 +11,31 @@ pub use self::level::{LevelFilter, ParseError as LevelParseError};
 #[cfg(feature = "env-filter")]
 pub use self::env::*;
 
+use std::sync::Arc;
+
 /// A `Layer` which filters spans and events based on a set of filter
 /// directives.
 ///
 /// **Note**: renamed to `EnvFilter` in 0.1.2; use that instead.
 #[cfg(feature = "env-filter")]
 #[deprecated(since = "0.1.2", note = "renamed to `EnvFilter`")]
 pub type Filter = EnvFilter;
+
+#[derive(Debug, Clone)]
+pub(crate) struct TargetFilter(Option<Arc<str>>);
```

Comment:
This shouldn't need to be `pub(crate)`; it's only used in this module...

Reply:
I can't because its returned here https://github.com/tokio-rs/tracing/blob/055c4c2465fa792015f45f3113a51b134729d1bf/tracing-subscriber/src/filter/env/directive.rs#L703